dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -134,13 +145,30 @@ private static Options configureOptions() {
     }
   }
 
-  static void configureMavenRepositories(CommandLine commandLine) {
+  static void configureMavenRepositories(CommandLine commandLine) throws ParseException {
```

Comment:
It would be better if this method were on `RepositoryUtility`. Then the non-final field could be private, and you could have a static method that returns it, so that we don't have a public static non-final field.